AlgoProject.UnitTests/SortingHandlerTest.cs

```diff
@@ -0,0 +1,66 @@
+using AlgoProject.Model;
+using AlgoProject.ViewModel.SortingTypes;
+using NUnit.Framework;
+using System;
+using Xunit;
+
+
+namespace AlgoProject.UnitTests
+{
+    public class SortingHandlerTest
+    {
+        private SortingHandler sortingHandler;
+
+        [SetUp]
+        public void Setup()
+        {
+            sortingHandler = new SortingHandler();
+        }
+
+        [Test]
+        public void GetClassSort_GoodExtension_ReturnBubbleSort()
+        {
+            //Arrange
+            var testBubbleSort = sortingHandler.GetClassSort(TypeSort.BubbleSort);
+
+            bool result = (testBubbleSort is BubbleSort);
+
+            NUnit.Framework.Assert.IsTrue(result);
+        }
+
+
+        [TestCase(TypeSort.MergeSort, false)]
```

Comment:
The test above seems to be doing the same, but for a BubbleSort. Can be merged into a single Test.

Reply:
This looks a bit strange. :)